src/components/PublicationRequestFormSections/FieldArrays/FundingFieldArray.js

```diff
@@ -20,6 +21,7 @@ const renderFunders = (fields, fundersValues, aspectFundedValues) => {
         <Row key={name} middle="xs">
           <Col xs={3}>
             <Field
+              autoFocus={!fields.value[index].aspectFunded}
```

Comment:
For consistency's sake we should key this off the id of the object not the value (sorry, it's a pedantic change)